src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -61,7 +61,7 @@ public function __construct($message, array $trace, $file)
         $line = $trace[$i];
         $this->triggeringFile = $file;
         if (isset($line['object']) || isset($line['class'])) {
-            if (isset($line['class']) && 0 === strpos($line['class'], SymfonyTestsListenerFor::class)) {
+            if (isset($line['class']) && is_subclass_of($line['class'], TestListener::class)) {
```

Comment:
this would now also apply for other PHPUnit TestListener implementations which have nothing to do with the Symfony deprecation helper. That's risky to me.

Reply:
Good point. Gone for a new approach that doesn't check the class at all.